src/components/Dashboard/FilingHistoryList/filings/ContinuationInApplication.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <FilingTemplate
+    class="continuation-in-application"
+    :filing="filing"
+    :index="index"
+  >
+    <template #subtitle>
+      <FiledAndPendingPaid
+        v-if="isFutureEffectivePending"
+        class="item-header-subtitle"
+        :filing="filing"
+        :index="index"
+      />
+
+      <FutureEffectivePaid
+        v-else-if="isFutureEffective"
+        class="item-header-subtitle"
+        :filing="filing"
+        :index="index"
+      />
+    </template>
+
+    <template #body>
+      <FutureEffectivePending
+        v-if="isFutureEffectivePending"
+        :filing="filing"
+      />
+
+      <FutureEffective
+        v-else-if="isFutureEffective"
+        :filing="filing"
+      />
+
+      <div
+        v-if="!!tempRegNumber && isStatusCompleted"
+        class="completed-ci-details"
+      >
+        <h4>Incorporation Complete</h4>
+
+        <p>
+          {{ companyName }} has been successfully continued in.
+        </p>
+
+        <p>
+          The system has completed processing your filing. You can now retrieve the business information.
+        </p>
+
+        <div class="reload-business-container text-center mt-6">
+          <v-btn
+            color="primary"
+            @click.stop="reloadWithBusinessId()"
+          >
+            <span>Retrieve Business Information</span>
+          </v-btn>
+        </div>
+      </div>
```

Comment:
Please show a screenshot of the above.

Also, please show a screenshot of the completed filing. I'm especially interesting if the court order info is displayed.

Reply:
![image](https://github.com/bcgov/business-filings-ui/assets/122495122/f89c8320-f6bd-4657-853b-44e2b5845b6b)